SECURITY.md

```diff
@@ -1,35 +0,0 @@
-<!-- BEGIN MICROSOFT SECURITY.MD V0.0.1 BLOCK -->
```

Comment:
Also @ibrahimhaddad do we need to include anything about LF secruity policy?

Reply:
Tracked here: https://github.com/recommenders-team/recommenders/issues/1997